src/raftstore/store/worker/apply.rs

```diff
@@ -166,53 +168,39 @@ pub enum ExecResult {
 }
 
 struct ApplyContext<'a> {
-    pub host: &'a CoprocessorHost,
-    pub wb: Option<WriteBatch>,
-    pub cbs: Vec<(Callback, RaftCmdResponse)>,
-    pub wb_last_bytes: u64,
-    pub wb_last_keys: u64,
-    pub sync_log: bool,
+    host: &'a CoprocessorHost,
+    wb: WriteBatch,
+    cbs: MustConsumeVec<(Callback, RaftCmdResponse)>,
```

Comment:
why do we need to check MustConsume?

Reply:
can we check this in the Context Drop? So we don't need to add a new struct.

Reply:
Because if `RaftCmdResponse` is leak, it may lead to dead lock in transaction layer. I want to move `wb` later, implement Drop will prevent moving any member fields.